docs/pages/admin-guides/access-controls/access-request-plugins/datadog-hosted.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Access Requests with Datadog Incident Management (Hosted)
+description: How to set up Teleport's Datadog Incident Management plugin for privilege elevation approvals.
+---
+
+## How it works
+
+With Teleport's Datadog Incident Management integration, engineers can access the
+infrastructure they need to resolve incidents without longstanding admin
+permissions that can become a vector for attacks.
+
+Teleport's Datadog Incident Management integration allows you to treat Teleport
+Role Access Requests as Datadog incidents, notify the appropriate on-call team,
+and approve or deny the requests via Teleport.
+
+This guide will explain how to set up Teleport's Access Request plugin for Datadog.
+
+![Datadog plugin architecture](../../../../img/enterprise/plugins/datadog/diagram.png)
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- A Datadog account with the role "Datadog Admin Role". The admin role is required to
+  create a Service Account and generate required credentials for the plugin.
+
+  You can see your role by visiting your user page in Datadog and navigating to
+  **Personal Settings -> Profile** and checking the value of the **Roles** field.
+
+- Either a Linux host or Kubernetes cluster where you will run the Datadog plugin.
+
+- (!docs/pages/includes/tctl.mdx!)
+
```

Comment:
Editor-like rights in Teleport are also required (ability to create roles and grant them to users).